src/openforms/formio/components/custom.py

```diff
@@ -189,6 +189,20 @@ def build_serializer_field(
 class Map(BasePlugin[Component]):
```

Comment:
it may be a good idea to define a `MapComponent` type that's more specific and includes the extra properties that are component specific.

Reply:
yeah definitely, seeing as this component will get a bit bigger with the other changes..
I've added a `MapComponent` type and replaced some instances of `Component` with `MapComponent`